staging/src/k8s.io/apimachinery/pkg/api/validate/each.go

```diff
@@ -149,7 +149,7 @@ func Unique[T any](_ context.Context, _ operation.Operation, fldPath *field.Path
 		// specify that just the keys should be rendered in the error.
 		errs = append(errs, field.Duplicate(fldPath.Index(i), val))
 	}
-	return errs
+	return errs.WithOrigin("unique")
```

Comment:
Do we need origin for these?  The Error already has `.Type = ErrorTypeDuplicate` -- similar to how required and forbidden don't really need an origin.

From tests:
    `field.Duplicate(field.NewPath("status", "reservedFor").Index(2), "").WithOrigin("unique")`
...is not really more informative than:
    `field.Duplicate(field.NewPath("status", "reservedFor").Index(2), "")`
...is it?